coast/TIDEGAUGE.py

```diff
@@ -130,17 +136,17 @@ def read_gesla_to_xarray_v3(cls, fn_gesla, date_start=None, date_end=None):
         -------
         xarray.Dataset object.
         """
-        debug(f'Reading "{fn_gesla}" as a GESLA file with {get_slug(cls)}')  # TODO Maybe include start/end dates
+        debug(f"Reading \"{fn_gesla}\" as a GESLA file with {get_slug(cls)}")  # TODO Maybe include start/end dates
         try:
             header_dict = cls.read_gesla_header_v3(fn_gesla)
             dataset = cls.read_gesla_data_v3(fn_gesla, date_start, date_end)
         except:
```

Comment:
We should avoid using bare excepts. These will catch **everything**. Including system exits.